x-pack/solutions/security/plugins/security_solution/public/attack_discovery/pages/results/attack_discovery_panel/tabs/alerts_tab/ai_for_soc/table.tsx

```diff
@@ -67,6 +67,7 @@ export interface TableProps {
  * It leverages a lot of configurations and constants from the Alert summary page alerts table, and renders the ResponseOps AlertsTable.
  */
 export const Table = memo(({ dataView, id, packages, query, ruleResponse }: TableProps) => {
+  console.log('//////////////////// rendering Table');
```

Comment:
don't forget to remove this :D

Reply:
Oops yup. I'm about to push a unit for the hook, I will remove as part of this commit. Thanks! 😆